azure-pipelines-external.yml

```diff
@@ -16,6 +16,25 @@ stages:
       - task: NuGetToolInstaller@1
         displayName: "Install NuGet"
 
+      - task: UseDotNet@2
+        inputs:
+          packageType: sdk
+          version: 5.0.100-preview.6.20318.15
+          installationPath: $(Agent.ToolsDirectory)/dotnet
+        displayName: "Install NET 5 SDK preview"
+
+      - task: UseDotNet@2
+        inputs:
+          packageType: sdk
+          version: 3.1.105
+          installationPath: $(Agent.ToolsDirectory)/dotnet
+        displayName: "Install dotnet 3.1 LTS"
+
+      - powershell: |
+          dotnet --list-sdks
+          dotnet --version
+        displayName: 'List dotnet versions and current version'
```

Comment:
as you can see [in the logs](https://sonarsource.visualstudio.com/DotNetTeam%20Project/_build/results?buildId=19586&view=logs&j=645a1db5-d210-5b98-34db-87d2a8fd8bb0&t=c5795919-3f3d-52d6-970e-8d4e2515eaaa), the latest (NET 5) version is the "current" version

That's fine, because our `VSBuild` step uses MSBuild. We don't use the `dotnet` tool anywhere.

The idea is that VS needs a compatible SDK installed on disk to be happy, hence we also install 3.1